doc/item_bonus.txt

```diff
@@ -198,6 +198,7 @@ bonus2 bSkillUseSPrate,sk,n;		Decreases SP consumption of skill sk by n%
 Atk/Def
 -------
 bonus2 bSkillAtk,sk,n;    		Increases damage of skill sk by n%
+bonus bSkillRatio,n;			Adds n to the skillratio of all attacks/skills that use it
```

Comment:
I don't know what to add here, but I had to read the whole commit and the pull request description to get all the info. I'm missing some infos in this doc

Reply:
I guess I could put the whole PR info in there but it doesn't really fit the "explained in a single line" style of this file.

Maybe need to have a separate section for it. /hm